lunavl/sdk/faceengine/engine.py

```diff
@@ -38,6 +39,9 @@ class VLFaceEngine:
         _faceEngine (PyIFaceEngine): python C++ binding on IFaceEngine, Root LUNA SDK object interface
     """
 
+    # path to a file with license info
+    license: Optional[Union[str, Path]] = None
```

Comment:
How it is supposed to be filled?

Reply:
User should set it before start application
```python
VLFaceEngine.license = "/etc/license.conf"
```

Reply:
m-key